manager/src/main/java/com/redhat/service/bridge/manager/BridgesServiceImpl.java

```diff
@@ -24,14 +27,24 @@
 import com.redhat.service.bridge.manager.api.models.responses.BridgeResponse;
 import com.redhat.service.bridge.manager.dao.BridgeDAO;
 import com.redhat.service.bridge.manager.models.Bridge;
+import com.redhat.service.bridge.rhoas.RhoasClient;
+import com.redhat.service.bridge.rhoas.dto.TopicAndServiceAccountRequest;
 
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Tag;
+import io.smallrye.mutiny.TimeoutException;
 
 @ApplicationScoped
 public class BridgesServiceImpl implements BridgesService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BridgesServiceImpl.class);
+    private static final String DEFAULT_RHOAS_TIMEOUT = "60";
+
+    @ConfigProperty(name = "event-bridge.manager.rhoas.enabled", defaultValue = "false")
+    boolean rhoasEnabled;
+
+    @ConfigProperty(name = "event-bridge.manager.rhoas.timeout", defaultValue = DEFAULT_RHOAS_TIMEOUT)
```

Comment:
@kostola Can we make the unit of the timeout explicit in the config variable e.g `event-bridge.manager.rhoas.timeout-seconds`. I think we can also put the default into the `application.properties`, so we either set it via ENV variable, or default to something sensible e.g:

```
event-bridge.manager.rhoas.timeout-seconds=${RHOAS_TIMEOUT_SECONDS:60}

Reply:
Ok but I'd like to discuss the naming first (see [my other comment](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/196#discussion_r778971192)).

Reply:
Replied.

Reply:
Done :+1: